src/main/scala/no/ndla/searchapi/controller/SearchController.scala

```diff
@@ -213,6 +213,8 @@ trait SearchController {
             asQueryParam(relevanceFilter),
             asQueryParam(contextFilters),
             asQueryParam(includeMissingResourceTypeGroup),
+            asQueryParam(embedResource),
+            asQueryParam(embedId)
```

Comment:
embedResource og embedId skulle altså være tilgjengelig i /group search? I så fall hadde jeg glemt å legge disse to linjene inn.

Reply:
Trur ikjke det er viktig i /group men skader ikkje å ha det der.